test/parallel/test-npm-install.js

```diff
@@ -29,11 +29,13 @@ const pkgContent = JSON.stringify({
 });
 
 const pkgPath = path.join(common.tmpDir, 'package.json');
-
 fs.writeFileSync(pkgPath, pkgContent);
 
 const proc = spawn(process.execPath, args, {
-  cwd: common.tmpDir
+  cwd: common.tmpDir,
+  env: {
+    path: [path.dirname(process.execPath), process.env.path].join(':')
```

Comment:
@Fishrock123 `process.execpath` will give the full path to the exec, not the bin directory. I've tried switching it up to make path simply `path.dirname(process.exectPath)` which should work and be cross platform